.gitmodules

```diff
@@ -23,15 +23,15 @@
 [submodule "mongo"]
 	path = projects/mongo
 	url = https://github.com/mongodb/mongo.git
-	branch = v7.0
+	branch = master
 [submodule "mongo-go-driver"]
 	path = projects/mongo-go-driver
 	url = https://github.com/mongodb/mongo-go-driver.git
 	branch = master
 [submodule "mongo-core-test"]
 	path = projects/mongo-core-test
-    url = https://github.com/mongodb-developer/mongo-core-test.git
-	branch = main
+    url = https://github.com/FerretDB/mongo-core-test.git
+	branch = command-args
```

Comment:
Change this after FerretDB/mongo-core-test#1

Reply:
Merged